_component_code/footer.md

```diff
@@ -9,3 +9,57 @@ collection: component_code
 ---
 
 {% include todo.html content="Set up code." %}
+
+# Footer
+
+The footer lives at the bottom of the page to help visitors find
+globally applicable, but non-primary navigation information. We
+recommend using it on every page and every wrapped application. For
+customer applications, it  should not exceed the articulated scope.
+
+## When to use footers
+
+Use the Rackspace footer on every application page to provide consistent
+global functionality on all of our platforms.
+
+## Best practices for footers
+
+Ensure the footer is placed at the bottom of the page, and flush to the right
```

Comment:
Place the footer at the bottom of every page. 

this is unclear to me: "flush to the right of any left-hand navigation, spanning the length of the screen." 
could you rephrase?

Reply:
@robb-romans - this *is* rephrased. I couldn't tell you what it meant either. @technabors halp